rero/rero-ils-ui
# autocomplete: fix suggestions are not displayed

The list of suggestions are sometime not displayed when the server take
few milliseconds to respond. This has been fixed in a new version of the
third party library `ngx-bootstrap`.
Moreover when a document is selected the search should be executed.

* Fixes the display of the suggestion list for unquick server response.
* Executes the search query when a document title suggestion is clicked.
* Closes rero/rero-ils#788.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Dependencies

- ng-core: https://github.com/rero/ng-core/pull/176

## Why are you opening this PR?

- To fix:  rero/rero-ils#788

## How to test?

- Go to the public and admin view.
- Type slowly a query, after 3 chars the list of suggestions should be displayed.
- If you select a document title the search should be executed without clicking the search button.

## Code review check list
